### DISCUSSION TITLE
Ability to open in specific browser based on which Windows desktop you open link in

### DISCUSSION THREAD

**Discussion by epiphanyplx** [nref/BrowseRouter#9] [Ideas]
I'm not sure if this is possible as I don't know if it's information Windows passes on, but it would be handy to be able to use specific browser based on which Windows desktop something is opened in.

(By Windows desktop I mean this: https://support.microsoft.com/en-us/windows/multiple-desktops-in-windows-36f52e38-5b4a-557b-2ff9-e1a60c976434#WindowsVersion=Windows_11)

I started using this feature recently to separate work/non-work computer use and it is actually what made me track this program down as while it's obnoxious to have a link open in incorrect browser, it's extra obnoxious to have link open in incorrect desktop.

**Comment by nref** [comment]
I definitely understand how obnoxious it is for a link or app to open in another desktop. *cough* sublime text...

I'll look into how we could add that. I'll get back to you!
